syfertext/vocab.py

```diff
@@ -18,15 +18,11 @@ def __init__(self, model_name: str):
 
         self.model_path = os.path.join(dirname, "SyferText", model_name)
 
-        # Create the 'strings' list that holds words that the Vocab object knows and
-        # have vectors for
-        strings = self.load_strings()
-
         # Create a `StringStore` object which acts like a lookup table
         # mapping between all strings known to the vocabulary and
         # their hashes. It can be used to retrieve a string given its hash
         # key, or vice versa.
```

Comment:
Would you please mention in this commit that only strings that are encountered will be stored here